apps/bfd-shared-test-utils/pom.xml

```diff
@@ -62,6 +62,12 @@
             <groupId>com.newrelic.telemetry</groupId>
             <artifactId>dropwizard-metrics-newrelic</artifactId>
         </dependency>
+        <dependency>
+            <!-- Used to run our unit and integration tests. -->
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>compile</scope>
```

Comment:
Why do we need junit in compile scope?

Reply:
Execellent question.  The reason it's needed here in compile scope is because I added some test assertions in FileBasedAssertionHelper.java.

The reason it's not a terrible idea to include junit in compile scope here is that this is a module that is used in test scope by our other modules.